.circleci/config.yml

```diff
@@ -13,27 +13,54 @@ e2eTest: &e2eTest
           wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
           chmod +x ./e2e-harness
 
-    - run: ./e2e-harness localkube
+    - run:
+        name: Get e2ectl binary
+        command: |
+          wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2ectl/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+          chmod +x ./e2ectl
 
     - run:
-        # Project specific: Only disable kube-dns in this project,
-        # since it cannot run side-by-side with core-dns.
-        name: Disable minikube kube-dns addon
-        command: sudo ./minikube-linux-amd64 addons disable kube-dns
+        name: Get kubectl binary
+        command: |
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
```

Comment:
Are we sure we want to always test against latest and not some predefined versioned?

Reply:
I think it's fine to always use the kubectl latest binary for now. We never had any big breaking changes in kubectl IIRC

Reply:
It's consistent with practice so far (see the definition of e2e-harness "runtime" part https://github.com/giantswarm/e2e-harness/blob/master/Dockerfile#L29)

Since newer kubectl is typically backward compatible with previous k8s releases this should be safe, with less maintenance overhead compared to case where kind and kubectl in e2e-harness would be kept in sync.

Reply:
Yes, we should be testing with a recent version of k8s so I think this is OK.